frontend/src/app/libs/event-description/components/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="breadcrumb">
+	<ng-container *ngFor="let breadcrumb of breadcrumbs; let i = index;">
+		<img *ngIf="i === 1" src="assets/images/icons/icon-chevron-right.svg" alt="icons" class="breadcrumb__img">
+		<span [routerLink]="breadcrumb.url" class="breadcrumb__link">
```

Comment:
Class attribute always should be the first attribute of element.
This is for good BEM readability.